### ISSUE TITLE
Accessibility Checker sample TODO

### ISSUE THREAD

**Issue by AnnaTomanek** [ckeditor/ckeditor-sdk#181]
Some tasks to do before we go online:

- [ ] Update `a11ychecker` and `balloonpanel` packages to the final builds.
- [x] Add some more tests/errors to the sample content to make it even better (@mlewand).

**Comment by mlewand** [comment]
## What more assessments can we use?

### `headersHaveText`

> All headers should contain readable text
> Users with screen readers use headings (e.g. <code>h1</code> elements), just like the tabs to navigate the structure of a page. All headings should contain either text, or images with appropriate alternative text.

We could simply add some header without text or with an image without a text. I've pushed a sample example to `t/181` branch. Unfortunately, currently we have no quick fix for it.

I've posted some notes about sample to #182, other than that everything looks fine.

----

There is one nice accessibility assessment `listOfLinksUseList`, but I don't like it's implementation. Let me put it's description:

> A list of links separated by non-readable characters should be in an ul or ol

The reason for this is that it's reporting each link, with exception of the last one, as an error. So it would bump error counts by 4 or so for a decent example, and that's not what we want. In addition to this I see that it reports also some false-positives.

**Comment by mlewand** [comment]
@AnnaTomanek any thoughts about `headersHaveText` assessment?

**Comment by AnnaTomanek** [comment]
One more useful test was `documentAbbrIsUsed` (we have WYSIWYG, WCAG and some more abbreviations - would be really fitting). This, however, seems to be buggy, too: https://github.com/cksource/quail/issues/20

**Comment by AnnaTomanek** [comment]
The `headersHaveText` test is a bit unclear to me - neither do I have a good example for it (I cannot imagine any reasonable person putting an image and no text into the header... and I only have one image for this sample so let's not lower the sample quality perhaps by putting inconsistent images? And an empty header would probably get removed by the editor anyway...) nor is the rule/explanation convincing. So with no quick fix for this, I'd probably vote against adding it.

Honestly if nothing else is a good match, we can stay with what we have.

**Comment by mlewand** [comment]
All right, that's fine. I also think that we're good with what we currently have. So the last thing remaining is to make a final AC build and link it in SDK.
